test/sol6/kyberNetwork.js

```diff
@@ -1060,7 +1060,8 @@ contract('KyberNetwork', function(accounts) {
                 });
 
                 // 2 types of trade that specific which reserve will be choosen
```

Comment:
if we are here.
can you fix the comment

seems it should have
specific --> specify

choosen --> chosen